src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -94,7 +97,7 @@ def get_attrs(self, item_list, user, **kwargs):
             if item.owner_id is not None:
                 owners_by_type[actor_type_to_string(item.owner.type)].append(item.owner_id)
 
-        resolved_actors = {}
+        resolved_actors = {}  # type: Dict[str, Dict[Optional[int], Optional[int]]]
```

Comment:
Why do you have to do this, why can't you actually type it like 
```suggestion
        resolved_actors: Dict[str, Dict[Optional[int], Optional[int]]] = {}
```

Reply:
Cause I googled my way through error messages